### DISCUSSION TITLE
[Package Request] AzCopy

### DISCUSSION THREAD

**Discussion by smajchrz** [microsoft/winget-pkgs#4826] [General] [answered]
It would be nice to have AzCopy in repo.

Not sure if it possible as it is delivered as zipped .exe file.

Latest version are distributed via links from following site:
https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10

**Comment by piraces** [comment]
Now installation of zipped files are not supported (https://github.com/microsoft/winget-cli/issues/140), so I guess we will have to wait to the winget-cli getting support for zip files.

**Reply by tebeco** [reply]
https://github.com/microsoft/winget-cli/issues/140
https://github.com/microsoft/winget-cli/pull/2500

it seems that it is supported now

**Comment by Trenly** [comment, marked as answer]
* #96201
